### PR TITLE
DCGAN_tensorflow

### PR DESCRIPTION
**Pull request by shvmshri**

### CODE REVIEW

**Comment by nirmal-suthar** [comment]
- [ ] rename the folder to DCGAN_TensorFlow
- [ ] remove all unused img files (images that are not used in readme)
- [ ] make folder named assets and move all display files into this folder
- [ ] move all utilities function in separate file utils.py
- [ ] rename MNIST_model.py to model.py

File: generative_models/DCGAN_Tensorflow-master/MNIST_model.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+from tensorflow.keras.layers import Conv2DTranspose,Conv2D
+from tensorflow.keras.models import Sequential, Model, load_model
+import numpy as np
+
+# Model of Generator
+
+def make_generator():
+  model=Sequential()
+
+  model.add(Dense(7*7*256,input_shape=(100,)))
```
Comment (nirmal-suthar) [inline generative_models/DCGAN_Tensorflow-master/MNIST_model.py]: add input shape to parser as encoding_dims

File: generative_models/DCGAN_Tensorflow-master/main.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+# Plotting generator losses and discriminator losses
+plt.figure(figsize=(10,5))
+plt.title("Generator and Discriminator Loss During Training")
+plt.plot(plo[0],label="G")
+plt.plot(plo[1],label="D")
+plt.xlabel("iterations")
+plt.ylabel("Loss")
+plt.legend()
+plt.savefig('loss_plot.png')
+plt.show()
```
Comment (nirmal-suthar) [inline generative_models/DCGAN_Tensorflow-master/main.py]: move this function to utils.py

File: generative_models/DCGAN_Tensorflow-master/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+## Sources
+1. [Deep Learning Part -II(CS7015),Lec-1](https://www.youtube.com/watch?v=1ju4qmdtRdY&t=142s)
+2. [Deep Learning Part -II(CS7015),Lec-2](https://www.youtube.com/watch?v=uyuYfTMHZM0)
+
+
+
+  
+
+
+
```
Comment (nirmal-suthar) [inline generative_models/DCGAN_Tensorflow-master/README.md]: remove stray lines

File: generative_models/DCGAN_Tensorflow-master/main.py [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+import numpy as np
+import os
+import PIL
+import matplotlib.pyplot as plt
+from tensorflow.keras.models import Sequential, Model, load_model
+from tensorflow.keras.optimizers import Adam
+from tensorflow.keras.losses import BinaryCrossentropy
+from IPython import display
+from MNIST_model import make_generator,make_discriminator
+
```
Comment (nirmal-suthar) [inline generative_models/DCGAN_Tensorflow-master/main.py]: use parser for passing hyperparameter like batch_size, learning rate, etc

File: generative_models/DCGAN_Tensorflow-master/main.py (hunk 2 above)
Comment (shvmshri) [reply generative_models/DCGAN_Tensorflow-master/main.py]: which files do i need to move in assets folder

Comment (nirmal-suthar) [reply generative_models/DCGAN_Tensorflow-master/main.py]: all display files that used in readme, should be in assets (like result, plot, etc.)
